bin/cheritest/cheritest.c

```diff
@@ -1936,6 +1970,7 @@ signal_handler_clear(int sig)
 static inline void
 set_thread_tracing(void)
 {
+#ifdef __mips__
```

Comment:
Should we wait for #479 to be merged?

Reply:
#479 is now merged, should use `CHERI_START_USER_TRACE`